package.json

```diff
@@ -18,7 +18,9 @@
     "jest": "jest --coverage test/jest/*.js",
     "predocs": "rimraf ./jsdocs && mkdir jsdocs/ && ./utils/make-jsdoc-readme.js > ./jsdocs/jsdoc-temp.md",
     "docs": "./node_modules/.bin/jsdoc src/ -c ./docs/config.json -d ./jsdocs/ -P ./package.json -R ./jsdocs/jsdoc-temp.md -r",
-    "test:integration": "nyc --reporter=text --reporter=html mocha test/it/*.js --no-timeouts",
+    "test:integration": "yarn test:integration:oauth && yarn test:integration:ssws",
+    "test:integration:ssws": "OKTA_CLIENT_AUTHORIZATIONMODE=SSWS nyc --reporter=text --reporter=html mocha test/it/*.js --no-timeouts",
+    "test:integration:oauth": "OKTA_CLIENT_AUTHORIZATIONMODE=PrivateKey nyc --reporter=text --reporter=html mocha test/it/user-get.js --no-timeouts",
```

Comment:
We will only run `user-get.js` test in `PrivateKey` mode.
This will ensure we have a e2e sanity test on this feature while also saving us build time.
Locally, we can technically run any test whose endpoints are open for oauth.